docs/core/my-first-transaction.md

```diff
@@ -591,7 +591,7 @@ Once you have executed your first transaction, you may refer to the document [Li
 
 ## Reference
 
-* [Welcome page](welcome-to-libra.md).
+* [Welcome page](/docs/welcome-to-libra).
```

Comment:
I am curious why you are making this change here and not to the others below it.